SpatialGDK/Source/SpatialGDK/Private/Utils/InterestFactory.cpp

```diff
@@ -545,13 +555,28 @@ QueryConstraint InterestFactory::CreateAlwaysInterestedConstraint(const AActor*
 	return AlwaysInterestedConstraint;
 }
 
-QueryConstraint InterestFactory::CreateAlwaysRelevantConstraint() const
+SpatialGDK::QueryConstraint InterestFactory::CreateGDKSnapshotEntitiesConstraint() const
 {
 	QueryConstraint AlwaysRelevantConstraint;
 
 	Worker_ComponentId ComponentIds[] = { SpatialConstants::STARTUP_ACTOR_MANAGER_COMPONENT_ID,
-										  SpatialConstants::VIRTUAL_WORKER_TRANSLATION_COMPONENT_ID,
-										  SpatialConstants::ALWAYS_RELEVANT_COMPONENT_ID };
+										  SpatialConstants::VIRTUAL_WORKER_TRANSLATION_COMPONENT_ID };
```

Comment:
Even you don't need it, worth adding one for the player spawner so we don't trip on it later on?